tests/components/switchbee/test_data.py

```diff
@@ -0,0 +1,141 @@
+"""Test data for the tests of the SwitchBee integration."""
```

Comment:
A better option would be to move this content to `__init__.py` and remove this file